### PR TITLE
Retrieve url of market endpoint from config file

### PR DESCRIPTION
**Pull request by alansemenov**

### CODE REVIEW

File: src/main/java/com/enonic/xp/app/main/MarketConfigImpl.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+public class MarketConfigImpl
+    implements MarketConfig
+{
+    private Configuration config;
+
+    @Activate
+    public void activate( final Map<String, String> map )
+    {
+        this.config = ConfigBuilder.create().
+            load( getClass(), "com.enonic.xp.market.cfg" ).
```
Comment (rymsha) [inline src/main/java/com/enonic/xp/app/main/MarketConfigImpl.java:22]: I don't think this will find a file in config folder.

Comment (alansemenov) [reply src/main/java/com/enonic/xp/app/main/MarketConfigImpl.java:22]: It fails before that... :(
image

Comment (rymsha) [reply src/main/java/com/enonic/xp/app/main/MarketConfigImpl.java:22]: This is actually after :)

File: src/main/java/com/enonic/xp/app/main/MarketConfigService.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.enonic.xp.app.main;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+
+@Component(immediate = true, service = MarketConfigService.class, configurationPid = "com.enonic.xp.market")
+public class MarketConfigService
+{
+
+    private String marketUrl;
```
Comment (rymsha) [inline src/main/java/com/enonic/xp/app/main/MarketConfigService.java:10]: final

**Comment by alansemenov** [comment]
Requires https://github.com/enonic/lib-admin-ui/pull/1398

**Pull request merged**